docs/source/access-data/tutorials.rst

```diff
@@ -1,7 +1,7 @@
 Tutorials
 ============
```

Comment:
It might be simpler to eliminate this page and move the subsections directly into the side bar.